test/cql-pytest/test_filtering.py

```diff
@@ -125,3 +125,16 @@ def test_filter_like_on_desc_column(cql, test_keyspace):
         cql.execute(f"INSERT INTO {table} (a, b) VALUES (1, 'one')")
         res = cql.execute(f"SELECT b FROM {table} WHERE b LIKE '%%%' ALLOW FILTERING")
         assert res.one().b == "one"
+
+# Test that the fact that a column is indexed does not cause us to fetch
+# incorrect results from a filtering query (issue #10300).
+def test_index_with_in_relation(scylla_only, cql, test_keyspace):
```

Comment:
Can you please consider referring to the Cassandra issue I mentioned, and perhaps even use cassandra_bug instead of scylla_only? I think it is likely that this feature will be added to Cassandra soon as well, and it's a good idea to remember this in case in the future someone will be curious (moreover, if the Cassandra issue is fixed, and this test doesn't start to pass on Cassandra, we'll need to start worrying).

Reply:
I got confused with this and a different discussion. I meant CASSANDRA-17198. But maybe that's relevant for this issue as well! Can't you please take a look?

Reply:
Well, CASSANDRA-17198 is specifically about LIKE so it's not the right issue. So never mind. If you can't easily find a Cassandra issue on this, let's just call it scylla_only and be done with it.